library/spdm_requester_lib/libspdm_req_challenge.c

```diff
@@ -5,6 +5,7 @@
 **/
 
 #include "internal/libspdm_requester_lib.h"
+#include <math.h>
```

Comment:
Code in /spdm_requester_lib or /spdm_responder_lib can only use C freestanding headers. And, since this operation is a power-of-two you can use bit-shifts.